cmd/zrok/util.go

```diff
@@ -13,3 +17,23 @@ func mustGetAdminAuth() runtime.ClientAuthInfoWriter {
 	}
 	return httptransport.APIKeyAuth("X-TOKEN", "header", adminToken)
 }
+
+func parseUrl(in string) (string, error) {
+	// parse port-only urls
+	if iv, err := strconv.ParseInt(in, 10, 0); err == nil {
+		return fmt.Sprintf("http://127.0.0.1:%d", iv), nil
```

Comment:
this should test for < 65535, no?

Reply:
i feel like it should tell me that it's inferred 'http'. for example if i zrok share 443, i would have inferred https

Reply:
I wouldn't expect it to infer HTTPS URL scheme if the arg is "443", but I would expect that if the arg was "https".